src/components/CompareResults/CommonGraph.tsx

```diff
@@ -39,6 +39,7 @@ function CommonGraph({
       },
     },
     responsive: true,
+    maintainAspectRatio: false,
```

Comment:
I think we need a max-height somewhere
see the behavior for the graphs in https://deploy-preview-867--mozilla-perfcompare.netlify.app/subtests-compare-results?baseRev=6237f52367ba70569588068ce9c00e2607bca802&baseRepo=mozilla-central&newRev=0bc4d1520de177e298ad4d051b99d0ac435e2b3c&newRepo=mozilla-central&framework=13&baseParentSignature=5273367&newParentSignature=5273367 for example